src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -12,6 +12,7 @@ CA1830 | Performance | Info | PreferTypedStringBuilderAppendOverloads, [Document
 CA1831 | Performance | Warning | UseAsSpanInsteadOfRangeIndexerAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1831)
 CA1832 | Performance | Info | UseAsSpanInsteadOfRangeIndexerAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1832)
 CA1833 | Performance | Info | UseAsSpanInsteadOfRangeIndexerAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1833)
+CA1834 | Performance | Info | PreferStringContainsOverIndexOf, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1834)
```

Comment:
You might want to rebase your branch because CA1834 is already being used by your previous analyzer.